app/assets/javascripts/discourse/app/routes/discourse.js

```diff
@@ -92,7 +92,10 @@ const DiscourseRoute = Route.extend({
   },
 
   isPoppedState(transition) {
-    return !transition._discourse_intercepted && !!transition.intent.url;
+    return (
+      (!transition._discourse_intercepted && !!transition.intent.url) ||
+      (transition && transition.sequence > 1 && transition.urlMethod)
```

Comment:
Why do we check we have a transition here and not on the line before? That doesn’t seem necessary